app/controllers/api/v1/decoded_auth_token.rb

```diff
@@ -1,5 +1,9 @@
-class Api::V1::DecodedAuthToken < HashWithIndifferentAccess
-  def expired?
-    self[:exp] <= Time.now.to_i
+module Api
+  module V1
+    class DecodedAuthToken < HashWithIndifferentAccess
+      def expired?
+        Time.at(self[:exp]) <= 5.minutes.from_now
```

Comment:
Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.